### DISCUSSION TITLE
Text contrast

### DISCUSSION THREAD

**Discussion by zazou-d96** [lardbit/nefarious#153] [Q&A] [answered]
While I am really liking Nefarious, it is a bit of a headache becaue the text is so subdued. Stylish maybe but a real issue for those like me with sub optimal vision. Search results,Watched/Wanted etc generate text that is difficult t read...Would have loved a high contrast option.

**Comment by lardbit** [comment]
That's helpful feedback.  I am less skilled at the visualization/aesthetics, but I think I can help.  I'll work on updating the search, wanted, and watching screens so the text has more of a contrast.  I'll keep you posted.

**Reply by zazou-d96** [reply]
Appreciated

**Comment by lardbit** [comment, marked as answer]
Ok, I updated the wanted/watching & media results pages to use a higher contrast.  Changes should roll out in a couple hours once it's built.

![image](https://user-images.githubusercontent.com/45122868/143685726-b4e56b2b-c573-470a-9788-1044af6e6d2e.png)

![image](https://user-images.githubusercontent.com/45122868/143685733-db8b803a-6887-466c-ac69-0bb4d45a162c.png)
